packages/voting-contract/src/lib.rs

```diff
@@ -105,6 +105,20 @@ where
         .set_data(cosmwasm_std::to_binary(&resp)?))
 }
 
+fn update_ballot(
```

Comment:
Maybe we use this on lines 95-96 as well?
(optional)

Reply:
There are two saves vs update and save in heler - so we are back again to one extra read.
Is that okay?

Reply:
Okay, proper helper worked fine.
Done.